_data/tools.yml

```
@@ -1697,3 +1697,15 @@
       url: https://docs.getzingdata.com/docs/setting-up-a-data-source/trino_setup/
     - urltext: Zing Data with Trino tutorial
       url: https://docs.starburst.io/data-consumer/clients/zingdata.html
+- name: Trino Guru
```

```suggestion
- name: Guru Base
```

to avoid using the Trino brand